src/player_display.cpp

```diff
@@ -1308,10 +1311,9 @@ void player::disp_info()
     ctxt.register_action( "HELP_KEYBINDINGS" );
     std::string action;
 
-    std::string help_msg = string_format( _( "Press %s for help." ),
-                                          ctxt.get_desc( "HELP_KEYBINDINGS" ) );
-    mvwprintz( w_tip, point( FULL_SCREEN_WIDTH - utf8_width( help_msg ), 0 ), c_light_red, help_msg );
-    help_msg.clear();
+    right_print( w_tip, 0, +4, c_white, string_format(
++                     _( "< [<color_yellow>%s</color>] keybindings >" ),
++                     ctxt.get_desc( "HELP_KEYBINDINGS" ) ) );
```

Comment:
```suggestion
                    ctxt.get_desc( "HELP_KEYBINDINGS" ) ) );
```
Or something alike, but the "+" should go away. Same on the line above.

Reply:
Thanks a lot!